core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.controller.spec.ts

```diff
@@ -142,7 +142,7 @@ describe('Add Answer Group Modal Controller', function() {
     expect($scope.isSelfLoopWithNoFeedback(outcome2)).toBe(false);
   });
 
-  it('should close modal when saving answer group response', function() {
+  it('should save answer group response when closing the modal', function() {
```

Comment:
This comment applies to some other places in your PR too. I didn't explicitly mention them, but this is a general comment. Could you fix this in the other places also?

Reply:
Can you please take a look at my replies in the previous comments about this?